File: server/TaskMaster/TaskMaster.DataWebApi/Models/UserAccessLevel.cs
using TaskMaster.DataAccessModule.Constants;
using TaskMaster.DataAccessModule.Models;

namespace TaskMaster.DataWebApi.Models
{
	/// <summary>
	/// Уровень доступа пользователя к доске.
	/// </summary>
	public class UserAccessLevel
	{
		/// <summary>
		/// Пользователь.
		/// </summary>
		public User User { get; set; }

		/// <summary>
		/// Уровень доступа.
		/// </summary>
		public AccessLevelType AccessLevel { get; set; }
	}
}

File: client/task-master/src/app/models/UserAccessLevel.ts
import { User } from './User';
import { AccessLevelType } from './enum/AccessLevelType';

// Уровень доступа пользователя
export class UserAccessLevel {
  // Пользователь
  user: User;
  // Уровень доступа
  accessLevel: AccessLevelType;
}
